scripts/build-deb.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
```

Comment:
You don't want to put this file in the `debian` directory? It is specific to debian, and I would prefer to avoid yet another directory for just a single file.

Reply:
ok. I though it would be useful to have a scripts directory to store all future scripts.
I will move it to the debian directory